jsontableschema/types/datetime.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
```

Comment:
Not sure I'd separate date and datetime into distinct modules - they have much in common.